src/components/NcAppContent/NcAppContent.vue

```diff
@@ -72,6 +72,7 @@ The list size must be between the min and the max width value.
 
 <template>
 	<main id="app-content-vue" class="app-content no-snapper" :class="{ 'app-content--has-list': hasList }">
+		<slot name="pageHeading" />
 		<template v-if="hasList">
```

Comment:
> By the way, shouldn't it have id="page-heading-level-1"?

"page-heading-level-1" is needed only for this function https://github.com/nextcloud/server/blob/master/core/src/OCP/accessibility.js#L31 which won't be called in any app which is using `NcAppContent.vue`

> In this case shouldn't it be a prop instead of a slot?

Ok, i will create a prop instead. Thank you!

Reply:
> "page-heading-level-1" is needed only for this function

There is this id attr in the screenshot at https://github.com/nextcloud/photos/pull/1727